README.md

```diff
@@ -11,7 +11,7 @@ Data for this project was provided in CSV format. This repo includes a preproces
 - Omit blank and infinite values
 - Optimize data by removing unused fields and rounding to sensible digits
 
-To generate the needed JSON files, download the five CSV data files from [Google Drive](https://drive.google.com/drive/folders/1hrGuRmQtx-gTepN_H9BEfcGmnl5MCeAK?usp=sharing) and store them in `scripts/input`. Then run the following commands to convert them to JSON:
+To generate the needed JSON files, download the five CSV data files from [Google Drive](https://drive.google.com/drive/folders/1hBjKEYzRPY7qQlbnqyMaYRMZSuMtIABB?usp=sharing) and store them in `scripts/input`. Then run the following commands to convert them to JSON:
```

Comment:
Not for this PR, but we probably will want to pull this reference out to a stand-along (non-Google) location after we launch to decouple changes.

Reply:
Is #80